### PR TITLE
Re-adjust filters value applied to PAF output...

### PR DESCRIPTION
**Pull request by rilango**
Re-adjust the value of min residues and min overlap length. Also add filter to
target query.

### CODE REVIEW

**Comment by mimaric** [comment]
@rilango I'm not sure what's the point for `Trigger build` commit (https://github.com/clara-parabricks/ClaraGenomicsAnalysis/pull/450/commits/825f0e429a9e8c859a0212d81678aa2fa3222be9).
If it was just so the tests restart you can simply add a `rerun tests` to a PR

**Comment by mimaric** [comment]
@edawson @vellamike I'll let you make the decision on this PR as it has a lot more to do with accuracy than the overall code structure

**Comment by rilango** [comment]
rerun tests

File: cudamapper/src/overlapper_triggered.cu [hunk 1]
```diff
@@ -163,7 +163,8 @@ struct FilterOverlapOp
 
         return ((overlap.num_residues_ >= min_residues) &&
                 ((overlap_length / overlap.num_residues_) < min_bases_per_residue) &&
-                (query_overlap_length > min_overlap_len) &&
+                (query_overlap_length >= min_overlap_len) &&
+                (target_overlap_length >= min_overlap_len) &&
```
[the remark this answers is not in the document]
Comment (rilango) [reply cudamapper/src/overlapper_triggered.cu:167]: The change are mainly to align this filter with the one in miniasm.

https://github.com/lh3/miniasm/blob/ce615d1d6b8678d38f2f9d27c9dccd944436ae75/hit.c#L52

File: cudamapper/src/application_parameters.hpp [hunk 2]
```diff
@@ -43,8 +43,8 @@ class ApplicationParameters
     int32_t target_index_size               = 30;    // t
     double filtering_parameter              = 1.0;   // F
     int32_t alignment_engines               = 0;     // a
-    int32_t min_residues                    = 10;    // r
-    int32_t min_overlap_len                 = 500;   // l
+    int32_t min_residues                    = 100;   // r
+    int32_t min_overlap_len                 = 2000;  // l
```
[the remark this answers is not in the document]
Comment (rilango) [reply cudamapper/src/application_parameters.hpp:47]: With this change it is same.

https://github.com/lh3/miniasm/blob/ce615d1d6b8678d38f2f9d27c9dccd944436ae75/common.c#L7

File: cudamapper/src/overlapper_triggered.cu (hunk 1 above)
[the remark this answers is not in the document]
Comment (edawson) [reply cudamapper/src/overlapper_triggered.cu:167]: I see these match those of miniasm. Approving.

File: cudamapper/src/application_parameters.hpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (edawson) [reply cudamapper/src/application_parameters.hpp:47]: Yep, looks the same.

**Pull request merged**